Resources/views/CRUD/edit_orm_many_association_script.html.twig

```diff
@@ -34,6 +34,10 @@ This code manage the many-to-[one|many] association field popup
         initialize_popup_{{ id }}();
 
         var target = jQuery(this);
+        
+        if (target.attr('href').length === 0 || target.attr('href') === '#') {
```

Comment:
@greg0ire @jontorrado  We're not trying to detect valid urls here, we're trying to detect if the link is an anchor.
An anchor points to the either the exact same url as the current page or the same url with a different hash.
So basically to do that you have two solutions, the long one:
```js
// HTMLElement -> boolean
const isAnchor = a => (
  a.tagName === 'A'
  && a.origin === location.origin
  && a.pathname === location.pathname
  && a.search === location.search
)
```
or the short one:
```js
// HTMLElement -> boolean
const isAnchor = a => {
  const href = (a.getAttribute('href') || '').trim()
  // notice the href[0] === '#' and not href === '#' 
  return a.tagName === 'A' && (!href || href[0] === '#')
}
```
So the shortest way gives us, with jQuery:
```js
// jQuery -> boolean
function isAnchor ($a) {
    var href = ($a.attr('href') || '').trim();
    return $a.is('a') && (!href || href[0] === '#');
}
```

As for the `href="javascript:alert(this)"` case, nobody should use that, so don't bother.